lib/tasks/generateThemeDesignerResources.js

```diff
@@ -241,12 +260,31 @@ export default async function({workspace, dependencies, options}) {
 	// Only for type "library". Type "theme-library" does not provide a namespace
 	// Also needs to be created in case a library does not have any themes (see bIgnore flag)
 	if (namespace) {
-		log.verbose(`Generating .theming for namespace ${namespace}`);
-		const libraryDotThemingResource = generateLibraryDotTheming({
-			namespace,
-			version,
-			hasThemes
-		});
+		let libraryDotThemingResource;
+
+		// Do not generate a .theming file for the sap.ui.core library
+		if (namespace === "sap/ui/core") {
+			// Check if the .theming file already exists
+			libraryDotThemingResource = await workspace.byPath(`/resources/${namespace}/.theming`);
```

Comment:
I also thought about this but eventually decided to keep the variable to not create multiple places where a namespace might need to be adjusted. This way, in case we want to e.g. open up this feature to other libraries, we would only need to adjust one line (if-clause), not any other part of the code. For that reason I kept it as it was.